src/cprt_gridmap_filters/src/PreserveCostInflationFilter.cpp

```diff
@@ -71,7 +71,7 @@ bool PreserveCostInflationFilter<T>::update(const T &mapIn, T &mapOut) {
     return false;
   }
 
-  mapOut.add(this->outputLayer_, 0.0);
+  mapOut.add(this->outputLayer_);
```

Comment:
Yeah this makes sense. It wasn't passing through the invalid spaces properly but this fixes that. Good to know.